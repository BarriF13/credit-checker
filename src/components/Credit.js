

import React, { Component } from 'react'

export class Credit extends Component {
  render() {
    return (
      <div>
        <h3>Bitcoin balance:</h3>
      </div>
    )
  }
}

export default Credit;
