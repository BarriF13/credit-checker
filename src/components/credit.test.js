import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

import React, { Component } from 'react';

import { shallow } from 'enzyme';
import { Credit } from './Credit'

describe('Credit', () => {
  const credit = shallow(<Credit />);
  it ('renders properly', ()=>{
    expect(credit).toMatchSnapshot();
  })
})