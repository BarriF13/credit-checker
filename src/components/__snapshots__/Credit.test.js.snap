// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Credit renders properly 1`] = `ShallowWrapper {}`;
